docs/3.0.0-beta.x/guides/secure-your-app.md

```diff
@@ -0,0 +1,62 @@
+# Secure your application
+
+In this guide we will see how you can secure your Strapi application by using a tiers provider.
+
+::: tip
+In this example we will use [Sqreen](https://sqreen.com).
+:::
+
+There [onboarding](https://my.sqreen.com/new-application#nodejs-agent) is really easy to follow and understand.
+
+## Install Sqreen
+
+Sqreen is an Application Security Management. That enable protections tailored to your stack, get unprecedented visibility into your security and scale it in production.
+
+You will have to install Sqreen node_module in your application.
+
+:::: tabs
+
+::: tab yarn
+`yarn add sqreen`
+:::
+
+::: tab npm
+`npm install qreen`
+:::
+
+::::
+
+## Start your application programmaticaly
+
+We will have to plug the Sqreen node_module in the file we use to start Strapi.
+
+To do so you will have to create a `server.js` file to be able to start our application by running `node server.js`.
```

Comment:
You should explain that sqreen requires to be loaded first so we need to create a file